transports/transport-rest/transport-rest-client/src/test/java/io/vertx/core/net/impl/ConnectionBase.java

```diff
@@ -0,0 +1,398 @@
+/*
```

Comment:
must use vertx LICENSE
and why copy this file?

Reply:
we mock some methods of ConnectionBase have changed to final, but mockito's the old version can't mock the final method, so copy this file.